pod/authentication/models.py

```diff
@@ -139,3 +139,15 @@ def create_groupsite_profile(sender, instance, created, **kwargs):
             msg += '\n%s' % traceback.format_exc()
             logger.error(msg)
             print(msg)
+
+
+class AccessGroup(models.Model):
+    display_name = models.CharField(
+        max_length=64, unique=True, blank=False, default="")
+    name = models.CharField(
```

Comment:
name or code name ? (in relation with display_name)

Reply:
Je n'ai pas bien compris la remarque. Il y a name qui représente le nom du groupe d'un point de vue logique et display_name qui représente son nom d'affichage

Reply:
ben en fait, tu mets display_name et name mais pour une certaine cohérence je proposais de préciser display_name et code_name

Reply:
Ok ça me va je fais ça

Reply:
Merci !